docs/contributing/contributing.rst

```diff
@@ -147,7 +147,7 @@ Test Environment Variables
 
 .. admonition:: Partial install and failing tests
 
-    Some tests may fail with an ImportError if the required extras are not installed. 
+    Some tests may fail with an ImportError if the required extras are not installed.
```

Comment:
My editor ended up formatting all lines in this file when saving to remove extra spaces. Lemme know if you like to revert these changes!